pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/file/FileIngestService.java

```diff
@@ -179,6 +186,8 @@ protected void deleteCompletedFiles() {
 
     @Override
     protected void doStart() {
+        metricPublisher.startAsync();
```

Comment:
Can we call related metricPublisher stop method in doStop?

Reply:
done.